src/components/MoveControl.js

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2019, Globo.com (https://github.com/globocom)
+ *
+ * License: MIT
+ */
+
+import React from "react";
+import icons from "../icons";
+import MegadraftBlock from "./MegadraftBlock";
+
+const Options = ({ onClickUp, onClickDown, id, disableUp, disableDown }) => {
+  const onPointerDown = e => {
+    e.preventDefault();
+    e.stopPropagation();
+  };
+  return (
+    <div className="options">
+      <button
+        data-testid={`swap-up-${id}`}
+        className="options__button options__button--up"
+        onClick={onClickUp}
+        disabled={disableUp}
+        {...{ onPointerDown }}
```

Comment:
I think that's better to be more explicit here
```js
{...{ onPointerDown }}
```
passing the prop itself `onPointerDown={onPointerDown}`

Reply:
Done!